src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -2525,15 +2772,19 @@ window.qBittorrent.DynamicTable ??= (() => {
                 const id = row.rowId;
                 const value = this.getRowValue(row);
 
-                if (window.qBittorrent.PropFiles.isDownloadCheckboxExists(id)) {
-                    window.qBittorrent.PropFiles.updateDownloadCheckbox(id, value);
-                }
-                else {
+                if (!td.firstElementChild) {
                     const treeImg = document.createElement("img");
                     treeImg.src = "images/L.gif";
                     treeImg.style.marginBottom = "-2px";
-                    td.append(treeImg, window.qBittorrent.PropFiles.createDownloadCheckbox(id, row.full_data.fileId, value));
+                    td.appendChild(treeImg);
```

Comment:
If you don't mind, please prefer `Element.append()` over `Node.appendChild()` (globally).
IMO `Element` is the most common level to operate on. `Node` is for some rare cases.
```suggestion
                    td.append(treeImg);
```